src/lib/utils/string.js

```diff
@@ -18,3 +18,28 @@ export const truncateMiddle = (string, maxLength = null, ellipsis = '...') => {
 
   return firstHalf + ellipsis + lastHalf
 }
+
+/*
+ * Decreases font-size of string based on the string lenght
+ *
+ * @param {string | number} text - the string or number that we want to reduce the size of
+ * @param {number} fontBaseSize - the default size of the text
+ * @param {number} decreaseThreshold - the character amount where the font starts shrinking
+ * @param {number} decreaseRate - how much the fontSize decreases per character
+ * @param {number} minFontSize - the minimum size this text can have
+ *
+ * @return string
+ */
+
+export const decreaseFontSizeBasedOnCharacterAmount = (
```

Comment:
1. shortify as `normalizeByLength`
2. move to `normalizeText` utility as named export